src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/UserDefinedImplicitConversions.cs

```diff
@@ -583,6 +583,12 @@ private static bool IsEncompassingImplicitConversionKind(ConversionKind kind)
                 case ConversionKind.PointerToInteger:
                 case ConversionKind.IntegerToPointer:
                 case ConversionKind.IntPtr:
+
+                case ConversionKind.ExplicitTupleLiteral:
+                case ConversionKind.ExplicitTuple:
+
+                case ConversionKind.StackAllocToPointerType:
```

Comment:
>ConversionKind.StackAllocToPointerType [](start = 21, length = 38)

It is not obvious why is this correct fix. Some explanation and perhaps a quote from relevant portion of language specification might help.

Reply:
There is no language spec as this is a C# 7.2 feature.
This is the up to date document: [https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.2/span-safety.md](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.2/span-safety.md)

About why is that the correct fix, I'll add a comment to explain.

---
In reply to: [183471406](https://github.com/dotnet/roslyn/pull/26248#discussion_r183471406) [](ancestors = 183471406)